### DISCUSSION TITLE
Dose open-webui support cloudflare workers ai?

### DISCUSSION THREAD

**Discussion by SeqCrafter** [open-webui/open-webui#1410] [Q&A]
as the title mentioned, dose the webui support cloudflare workers ai:
https://developers.cloudflare.com/workers-ai/models/

**Comment by SeqCrafter** [comment]
Cloudflare AI has a lot of free and open sources LLM models, It's a good idea to support that.

**Comment by justinh-rahb** [comment]
Cloudflare Workers AI should already supported via our LiteLLM integration, though may require custom modified config.yaml

**Reply by justinh-rahb** [reply]
https://litellm.vercel.app/docs/providers/cloudflare_workers
